openwhisk/README.md

```diff
@@ -0,0 +1,260 @@
+# OpenWhisk Runtimes for Knative
+
+This directory contains Tekton Pipelines which can be used to Build and Serve Knative compatible applications (i.e., OpenWhisk Actions) on Kubernetes.
```

Comment:
Is this what this aims to be eventually?

I think the README should match the current of the `openwhisk` folder. 
At the moment there is a Task that enriches a Docker file and triggers the build via kaniko.

The E2E building the code and serving the app in Knative serving is provided via the runtime specific README, but there's no Task or Pipeline defined yet for that.

Reply:
👍